client/gutenberg/editor/calypsoify-iframe.jsx

```diff
@@ -239,6 +287,13 @@ class CalypsoifyIframe extends Component {
 					/>
 				</MediaLibrarySelectedData>
 				<EditorRevisionsDialog loadRevision={ this.loadRevision } />
+				<WebPreview
```

Comment:
Same comment I made in https://github.com/Automattic/wp-calypso/pull/31596/files#r267514774, since it's looking like a pattern; aren't we loading code we'll often never use? Can we avoid doing this?

Reply:
Well, we are only loading the `WebPreview` container. Its content should only be loaded and rendered when the `showPreview` prop is true.

In any case, it was the same in Classic Editor:
https://github.com/Automattic/wp-calypso/blob/f6b317ffe8145325afdd16b8a511c4e32c1ae52d/client/post-editor/post-editor.jsx#L380-L393